inc/components/components/post-meta/component.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "irving/post-meta",
+  "_alias": "irving/fragment",
+  "description": "Provide context for a specific post meta value.",
+  "config": {
+    "meta_key": {
+      "hidden": false,
+      "type": "string"
+    },
+    "meta_value": {
+      "hidden": false,
+      "type": "string"
```

Comment:
Even though numbers get converted to strings, meta values could also be arrays, so we probably don't want to enforce a type here.

Reply:
I wasn't sure if the type was optional. If I just leave it out, will the plugin just skip type verification/coercion?